azure-blob-nio/README.md

```diff
@@ -0,0 +1,333 @@
+# Azure Storage Blob NIO FileSystemProvider
```

Comment:
I'm not sure it makes sense to include the original README, CHANGELOG, and design doc in here. Maybe remove and add a new README that explains what we're doing and links out to the repo?

Reply:
Fair, I wasn't sure we'd keep it, but I had been thinking the initial copy should be pretty much the same, but I think I agree cleaning up irrelevant documentation won't break the bank.